api/v1alpha1/rolebindingtemplate_types.go

```diff
@@ -80,7 +80,8 @@ type RoleBindingTemplateScopes struct {
 
 // ScopeSelector wraps a k8s label selector
 type ScopeSelector struct {
-	MatchLabels map[string]string `json:"matchLabels,omitempty"`
+	MatchLabels      map[string]string                 `json:"matchLabels,omitempty"`
+	MatchExpressions []metav1.LabelSelectorRequirement `json:"matchExpressions,omitempty"`
```

Comment:
We need to make sure to roll out the CRD on installations before/at the same time as the operator :)